internal/pkg/externalplugins/cherrypicker/cherrypicker.go

```diff
@@ -75,6 +84,8 @@ type githubClient interface {
 	ListIssueComments(org, repo string, number int) ([]github.IssueComment, error)
 	GetIssueLabels(org, repo string, number int) ([]github.Label, error)
 	ListOrgMembers(org, role string) ([]github.TeamMember, error)
+	IsCollaborator(org, repo, user string) (bool, error)
+	AddCollaborator(org, repo, user, permission string) error // pull, triage, push, maintain, admin
```

Comment:
I think we can use [`RepoPermissions`](https://github.com/kubernetes/test-infra/blob/02c787924618a7f1aa8b1a4af92c43f9712fa6d4/prow/github/types.go#L467).

https://docs.github.com/en/rest/collaborators/collaborators#add-a-repository-collaborator

Reply:
I think it's better to using `RepoPermissionLevel`, only a permission role can be set.